converter/converter/pom.xml

```diff
@@ -91,6 +91,9 @@
                             *
                         </Import-Package>
                         <!--Include-Resource>{maven-resources},OSGI-OPT/src=src/main/java</Include-Resource-->
+                        <Provide-Capability>
+                            osgi.service;objectClass:List&lt;String&gt;="org.osgi.util.converter.Converter"
```

Comment:
Actually, the Converter does **not** provider a service. It did at some point in the past, but nowadays you'll obtain the standard converter by calling Converters.standardConverter();
You can still register your own (customized) Converter in the service registry, which I think will be the more common case, but that's outside of the converter bundle.

Reply:
Hi @bosschaert. Yeah, I realized this when I added the capability, but the problem is that if I take it out, then it does not resolve anymore. Any suggestions for an alternative to help it resolve properly even without a capability? Or maybe some other capability I can provide to give the resolver a hint?

Reply:
That Require-Capability for the converter service in the serializer and schematizer is incorrect and should simply be removed. The resolution should go via the the package. Since org.osgi.util.converter contains everything (including Converters.standardConverter()) importing that package should be enough... Since they already use the package that import should already be there so just removeing the Require-Capability should work.

Reply:
Of course, you are right. My resolution problem had a different cause, which I was finally able to track down.